### PR TITLE
updated engine docs and bumped default version to 7.0

### PR DESCRIPTION
**Pull request by sanderson**

### CODE REVIEW

File: doc/advanced-php-config.md [hunk 1]
```diff
@@ -4,158 +4,156 @@ This engine exposes configuration options through the [boxfile.yml](http://docs.
...
-  newrelic_webtransaction_name_remove_trailing_path: false
+run.config:
+  engine: php
+  engine.config:
+    # Web Server Settings
+    webserver: 'apache'
+    document_root: '/'
+
+    # PHP Settings
+    runtime: 'php-5.6'
```
Comment (danhunsaker) [inline doc/advanced-php-config.md]: 7.0?

Comment (sanderson) [inline doc/advanced-php-config.md]: Nice catch

File: doc/advanced-php-config.md [hunk 2]
```diff
@@ -250,6 +250,7 @@ Specifies which PHP runtime and version to use. The following runtimes are avail
 
 ```yaml
 run.config:
+  engine: php
   engine.config:
     runtime: 'php-5.6'
```
Comment (sanderson) [inline doc/advanced-php-config.md]: Nice catch

File: README.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ run.config:
 
 ## Basic Configuration
 
-This engine exposes configuration options through the [boxfile,yml](http://docs.nanobox.io/boxfile/), a yaml config file used to provision and configure your app's platform. 
+This engine exposes configuration options through the [boxfile,yml](http://docs.nanobox.io/boxfile/), a yaml config file used to provision and configure your app's platform.
```
Comment (sanderson) [inline README.md]: This was a double catch. It should actually be `boxfile.yml`

**Pull request merged**